modules/qna/src/backend/storage.ts

```diff
@@ -236,6 +236,20 @@ export default class Storage {
     return _.uniq(_.filter(allAnswers as string[], x => _.isString(x) && x.startsWith('#!')))
   }
 
+  async getCountByTopic(): Promise<{ [context: string]: number }> {
+    const qnas = await this.fetchQNAs()
+
+    const contexts = _.chain(qnas)
+      .flatMap(i => i.data.contexts)
+      .uniq()
+      .value()
+
+    return contexts.reduce((acc, context) => {
```

Comment:
Maybe use https://lodash.com/docs/4.17.15#countBy

Reply:
Good point, I over-complicated this for nothing